mistakedetection.tests/src/ca/mcgill/sel/mistakedetection/tests/MistakeDetectionPerformanceAnalysis.java

```diff
@@ -264,6 +342,298 @@ public void testStudentSolution20() {
+      var studElem = getConcatNames(m.getStudentElementNames());
+      
+      var mistakeType = m.getMistakeType().getName();
+      for (Mistake m1 : extraMistakes) {
+        if (m != m1) {
+          var instElem1 = getConcatNames(m1.getInstructorElementNames());    
+          var studElem1 = getConcatNames(m1.getStudentElementNames());
+          if (instElem.equals(instElem1) && studElem.equals(studElem1)
+              && !(m.getMistakeType().getName().equals(WROMG_MULTIPLICTY)
+                  && m1.getMistakeType().getName().equals(WROMG_ROLE_NAME))
+              && !(m.getMistakeType().getName().equals(WROMG_ROLE_NAME)
+                  && m1.getMistakeType().getName().equals(WROMG_MULTIPLICTY))) {
+            mistakeType += ", " + m1.getMistakeType().getName();
+            count++;
+          }
+        }
+      }
+      var fnlString = instElem + studElem;
+      if (!rendered.contains(fnlString) || mistakeType.equals(WROMG_ROLE_NAME) || mistakeType.equals(WROMG_MULTIPLICTY)) {
+        studentData.put(String.valueOf(id),
+            new Object[] {instElem, studElem, mistakeType, "", count, "=D" + id + "=E" + id, ""});
+        id++;
+        rendered.add(fnlString);
+      }
+    }
+
+    int nId = id;
+    studentData.put(String.valueOf(nId++), new Object[] {"", "", "", "", "", "", ""});
+    studentData.put(String.valueOf(nId++), new Object[] {"", "", "", "", "", "", ""});
+    studentData.put(String.valueOf(nId++),
+        new Object[] {"Total Mistakes", "", "", "=SUM(D2:D" + --id + ")", "=SUM(E2:E" + id + ")", "", ""});
+
+    studentData.put(String.valueOf(nId++),
+        new Object[] {"MDS vs Actual Decision", "", "", "", "", "=COUNTIF(F2:F" + id + ",\"False\")", ""});
+    
+    // Important for formula calculations
+    var totalMistakeindex = id + 3;
+    var MDSvsActualIndex = totalMistakeindex + 1;
+    var totalInstElemsIndex = MDSvsActualIndex + 1;
+    var instElemsIndex = totalInstElemsIndex + 1;
+    var studExtraIndex = instElemsIndex + 1;
+    var addFalseNegIndex = studExtraIndex + 1;
+    var addFalsePosIndex = addFalseNegIndex + 1;
+    var totalElemsIndex = addFalsePosIndex + 1;
+    var totalNumbVerdictsIndex = totalElemsIndex + 1;
+    var tnIndex = totalNumbVerdictsIndex + 6;
+    var tpIndex = tnIndex + 1;
+    var fpIndex = tpIndex + 1;
+    var fnIndex = fpIndex + 1;
+    var recallIndex = fnIndex + 1;
+    var precisionIndex = recallIndex + 1;
+    studentData.put(String.valueOf(nId++), new Object[] {"Total number of instructor elements", 102});
+    studentData.put(String.valueOf(nId++),
+        new Object[] {"Instructor elements present", "=COUNT(E2:E" + instructorElems + ")"});
+    studentData.put(String.valueOf(nId++),
+        new Object[] {"Extra Elements", "=SUM(IF(D" + extraElemsStart + ":D" + id + ">E" + extraElemsStart + ":E" + id
+            + ",D" + extraElemsStart + ":D" + id + ",E" + extraElemsStart + ":E" + id + "))"});
```

Comment:
The cell at this point should be marked as a `FormulaType.ARRAY` so it can be opened in spreadsheet programs without issue

Reply:
Tried this but got error cell.setCellFormula(FormulaType.ARRAY.toString);

Reply:
I also tried using `cell.getCTCell().getF().setT(STCellFormulaType.ARRAY)` but this did not work either